Gemfile

```diff
@@ -31,7 +31,7 @@ group :test do
 end
 
 group :production do
-  gem 'pg',             '0.17.1'
+  gem 'mysql2'
```

Comment:
if we 'll use heroku as production
then we should leave pg gem
@freiwillen - please confirm